tests/e2e/accounts/multisig/utils.go

```diff
@@ -0,0 +1,65 @@
+package multisig
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+	"google.golang.org/protobuf/runtime/protoiface"
+
+	"cosmossdk.io/simapp"
+	"cosmossdk.io/x/bank/testutil"
+
+	"github.com/cosmos/cosmos-sdk/crypto/keys/secp256k1"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+)
+
+var members = []sdk.AccAddress{}
+
+type ProtoMsg = protoiface.MessageV1
+
+func init() {
+	for i := 0; i < 10; i++ {
+		addr := secp256k1.GenPrivKey().PubKey().Address()
+		members = append(members, sdk.AccAddress(addr))
+	}
+}
+
+type E2ETestSuite struct {
+	suite.Suite
+
+	app *simapp.SimApp
+}
+
+func NewE2ETestSuite() *E2ETestSuite {
+	return &E2ETestSuite{}
+}
+
+func (s *E2ETestSuite) SetupSuite() {
+	s.T().Log("setting up e2e test suite")
```

Comment:
nit: do you need this for debugging? I would avoid this extra output here and in L44.